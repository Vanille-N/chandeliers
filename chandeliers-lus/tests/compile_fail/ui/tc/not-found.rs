//! Constant or local variable not found.
chandeliers_lus::decl! {
    const Y : int = 42;
    const Z : int = 12;
    node foo() returns (i : int);
    let
        i = X;
    tel
}

fn main() {}
