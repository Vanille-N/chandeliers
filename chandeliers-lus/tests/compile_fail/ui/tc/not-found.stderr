error: Variable X not found in the context.
 --> tests/compile_fail/ui/tc/not-found.rs:6:13
  |
6 |         i = X;
  |             ^
  |
  = note: Perhaps you meant one of the local variables: i
  = note: or one of the global variables: Y, Z
